DevProxy.Abstractions/Utils/MSGraphDbUtils.cs

```diff
@@ -9,17 +9,18 @@
 
 namespace DevProxy.Abstractions.Utils;
 
-public static class MSGraphDbUtils
+public sealed class MSGraphDbUtils(HttpClient httpClient, ILogger<MSGraphDbUtils> logger) : IDisposable

```

Comment:
Since we're converting this into an actual class, let's rename the class to `MSGraphDb` and drop `Utils`.

Reply:
renamed. 
The class is in Utils folder, it suggests itself to move it somewhere else.